packages/python-packages/api-stub-generator/apistub/nodes/_function_node.py

```diff
@@ -12,9 +12,20 @@
 KW_ARG_NAME = "**kwargs"
 VALIDATION_REQUIRED_DUNDER = ["__init__",]
 KWARG_NOT_REQUIRED_METHODS = ["close",]
-TYPEHINT_NOT_REQUIRED_METHODS = ["close",]
-REGEX_ITEM_PAGED = "~[\w.]*\.([\w]*)\s?[\[\(][\w]*[\]\)]"
+TYPEHINT_NOT_REQUIRED_METHODS = ["close", "__init__"]
+REGEX_ITEM_PAGED = "~[\w.]*\.([\w]*)\s?[\[\(][^\n]*[\]\)]"
```

Comment:
Making certain I grok this, you're moving to explicitly `include everything up to the newline` because otherwise you were just eating comments right?

Reply:
REGEX for item paged was finding only part of return type if type is like ~azure..ItemPaged[someType[SomeSubType]]. Previous REGEX stops scan as soon as it find closing bracket.